### ISSUE TITLE
Update `missing-playwright-await` rule to warn about unnecessary awaits

### ISSUE THREAD

**Issue by AlexDaniel** [playwright-community/eslint-plugin-playwright#70]
Bad:
```js
await page.locator('.hello-world')
```
The code above is wrong because it doesn't do anything. We just await a locator and then throw it away, but that doesn't test anything.

Good:
```js
await expect(page.locator('.hello-world')).toBeVisible()
```
or
```js
await page.locator('.hello-world').waitFor()
```
or anything else, as long as you actually use the locator for something.

**Comment by carmanchris31** [comment]
📝  note:This can be achieved with [@typescript-eslint/await-thenable](https://typescript-eslint.io/rules/await-thenable):

![Screen Shot 2023-01-04 at 5 32 44 PM](https://user-images.githubusercontent.com/9935648/210681465-1f98c861-8442-4765-a8e2-093165de54f4.png)

**Comment by mskelton** [comment]
That rule does require type information which does require additional configuration and it does impact performance.  I would like to see this rule still be implemented given that it's a common mistake that would be nice to just have working automatically for everyone.
